data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+			choice
+				`	"Of course, what do you need me to do?"`
+					goto action
+				`	"This seems like a bad idea, ask someone else."`
+					decline
+			
+			label brethren
+			`	"Then you have heard about the stories of the Drak modifying our people..." He seems a bit sad. "I do not believe in such things, what makes a Hai -or any other species- “True” is in his heart. It will take him wherever he wants to go." He pauses a bit, thinking, and when he speaks again it is with deep conviction. "Even if we war with the Hai government it does not mean we hate it's people or them us! We do it only because we must, in order to keep living our way. We can still communicate with our people, and eventually they will all be freed from the limits of the Hai government."`
+			choice
+				`	"Will this get me into any trouble with the Hai?"`
+				`	"What limits?"`
+					goto limits
+				`	"Why do you need me to do it anyway?"`
+					goto whyme
+				`	"I believe they would not have their place here, I will not bring them."`
+					decline
+			`	"Quite the contrary! They may not believe in our way of doing things, but they would never dare to restrict the movement of their own people. It would be against all they stand for. Even if we behave differently they know we are still Hai, and that we would never hurt Hai civilians."	He seems confident.`
+			choice
+				`	"What is it exactly you want me to do?"`
+					goto action
+				`	"I am not interested in moving people around."`
+					decline
+			
+			label limits
+			`	"I mean that they are prevented from reaching their true potential. Once we had many stars and planets to call ours, but since then most of the Hai have decided to stop changing, to stay peaceful with all species, no matter how primitive. They have chosen to decay, to grow soft. The Unfettered are the ones who choose to keep our legacy, our way of life and then to evolve, to grow."`
+			choice
+				`	"What is it exactly you want me to do?"`
+				`	"I am not interested in moving people around."`
+					decline
+			
+			label action
+			`	"Oh it is pretty simple actually, you only need to go to <stopovers>, pick up <bunks> Hai and bring them to this planet, they will find their way from there." You somehow doubt that affirmation, seeing how everything here is in a deprecated state to say the least. You hope that at least the Hai actually know about that fact, or you may actually have to bring them back...`
```

Comment:
"deprecated" - not the word you're looking for

Reply:
"Decadent" would be it?
I am still learning the nuances of english, being able to understand others or to convey my general meaning is of course not really sufficient when it comes to writing, which is why I need the help of you guys.
Btw, thx for the review!

Reply:
My guess would have been 'dilapidated" - but I'm not entirely sure the vibe you're looking for.

Reply:
What I mean is that they live in a city that would almost look abandoned, with living conditions that aren't the best

Reply:
Dilapidated would be best then I think.